package.json

```diff
@@ -5,7 +5,9 @@
   "main": "localdev.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "./node_modules/.bin/semistandard *.js"
+    "lint": "./node_modules/.bin/semistandard *.js",
+    "start": "./localdev.js run zen",
+    "gulp": "gulp dev --gulpfile"
```

Comment:
One would expect `npm run gulp` to just run `gulp`, but it doesn't. Would rather `npm run gulp` only match to just `gulp`. The `dev --gulpfile` part should probably be specified by run.js, like it is now.